tests/jeos/firstrun.pm

```diff
@@ -119,6 +119,22 @@ sub verify_partition_label {
     script_output('sfdisk -l') =~ m/Disklabel type:\s+$label/ or die "Wrong partion label found, expected '$label'";
 }
 
+sub verify_selinux {
+    if (has_selinux_by_default) {
+        validate_script_output("sestatus", sub { m/SELinux status: .*enabled/ });
+    } else {
+        my $selinux_supported = script_run("grep -qw selinux /sys/kernel/security/lsm");
```

Comment:
```suggestion
        my $selinux_supported = script_run("grep -qw selinux /sys/kernel/security/lsm") == 0;
```

Otherwise the variable has the opposite meaning to it's name.

Reply:
Good catch, thanks! Fixed in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20693/commits/1b8b2d3d5569c0e9503b8f6f0d4b1d9b5c0d1e55